test/module/irohad/multi_sig_transactions/transport_test.cpp

```diff
@@ -59,6 +101,20 @@ class TransportTest : public ::testing::Test {
   std::shared_ptr<iroha::DefaultCompleter> completer_;
   std::shared_ptr<iroha::MockMstTransportNotification>
       mst_notification_transport_;
+  std::unique_ptr<shared_model::validation::MockValidator<
+      shared_model::interface::Transaction>>
+      interface_tx_validator;
+  std::unique_ptr<
+      shared_model::validation::MockValidator<iroha::protocol::Transaction>>
+      proto_tx_validator;
+  std::shared_ptr<shared_model::proto::ProtoTransportFactory<
+      shared_model::interface::Transaction,
+      shared_model::proto::Transaction>>
+      tx_factory;
+  std::shared_ptr<MstTransportGrpc> transport;
+  std::shared_ptr<shared_model::interface::Peer> peer;
+  // stub will be deleted by unique_ptr created in client_creator
+  iroha::network::transport::MockMstTransportGrpcStub *stub;
```

Comment:
Again, codestyle. Here and forth
```suggestion
  iroha::network::transport::MockMstTransportGrpcStub *stub_;
```
https://google.github.io/styleguide/cppguide.html#Variable_Names

Reply:
This field is public.